AK/Utf16View.cpp

```diff
@@ -133,6 +133,12 @@ ErrorOr<String> Utf16View::to_utf8(AllowInvalidCodeUnits allow_invalid_code_unit
     return builder.to_string();
 }
 
+bool Utf16View::is_ascii() const
+{
+    // FIXME: Petition simdutf to add an ASCII validator for UTF-16.
```

Comment:
Perhaps now is the time to open an issue in their repo?

Reply:
actually...we can avoid another simdutf API altogether now that I think about it:
```c++
simdutf::validate_ascii(reinterpret_cast<char const*>(m_string), length_in_code_units() * 2);
```

because every other byte would be 0, which is ASCII. We do similar "treat these char16s as bytes pls" elsewhere.